platform-includes/getting-started-config/java.log4j2.mdx

```diff
@@ -6,36 +6,37 @@ The `ConsoleAppender` is provided only as an example of a non-Sentry appender se
 
 </Note>
 
-
 ```xml
 <?xml version="1.0" encoding="UTF-8"?>
-<Configuration status="warn" packages="org.apache.logging.log4j.core,io.sentry.log4j2">
-    <Appenders>
-        <Console name="Console" target="SYSTEM_OUT">
-            <PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n"/>
-        </Console>
-        <Sentry name="Sentry"
-                dsn="___PUBLIC_DSN___" />
-    </Appenders>
-    <Loggers>
-        <Root level="info">
-            <AppenderRef ref="Sentry"/>
-            <AppenderRef ref="Console"/>
-        </Root>
-    </Loggers>
+<Configuration xmlns="https://logging.apache.org/xml/ns"
+               xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+               xsi:schemaLocation="https://logging.apache.org/xml/ns https://logging.apache.org/xml/ns/log4j-config-2.xsd">
+
+  <Appenders>
+    <Console name="CONSOLE" target="SYSTEM_OUT">
+      <PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n"/>
+    </Console>
+    <Sentry name="SENTRY" dsn="___PUBLIC_DSN___"/>
+  </Appenders>
+
+  <Loggers>
+    <Root level="INFO">
+      <AppenderRef ref="CONSOLE"/>
+      <AppenderRef ref="SENTRY" level="ERROR"/>
```

Comment:
Is there a particular reason for adding the `level ` here, compared to the original?
Do you want to highlight the possibility of overriding the level like in `docs/platforms/java/common/legacy/log4j2/index.mdx`? If so, I would suggest to also add a comment here, like in the mentioned file.

Reply:
@lbloder, earlier the snippet was as follows:

```xml
<Root level="info">
  <AppenderRef ref="Sentry"/>
  <AppenderRef ref="Console"/>
</Root>
```

Though this didn't capture the configuration stated at the beginning of the `java.log4j2.mdx` file:

> The following example using the `log4j2.xml` format to configure a `ConsoleAppender` that logs to standard out at the `INFO` level, and a `SentryAppender` that logs to the Sentry server at the `ERROR` level.

That is, sending `ERROR`s (and of higher severity) to Sentry was misconfigured – I fixed it.

> Do you want to highlight the possibility of overriding the level like in `docs/platforms/java/common/legacy/log4j2/index.mdx`...

I am not able to follow you. I fixed `legacy/log4j2/index.mdx` too, since that document was stating:

> The following example configures a `ConsoleAppender` that logs to standard out at the `INFO` level and a `SentryAppender` that logs to the Sentry server at the `WARN` level.

Hence, AFAIC, all configurations now match what is stated to be expected from them.

Reply:
@vy Sorry for the confusion.
Thank you, for fixing this, I just looked at the differences in the xml and thus overlooked the fact that the description was wrong in the first place.

>I am not able to follow you. I fixed legacy/log4j2/index.mdx too, since that document was stating
I was referring to the inline comment above the `AppenderRef` that might be helpful for users copying the xml.
For example:
```
<Root level="info">
    <AppenderRef ref="CONSOLE"/>
		<!-- Note that the Sentry logging threshold is overridden to the ERROR level -->
    <AppenderRef ref="SENTRY" level="ERROR"/>
</Root>
```